src/lib/invention/groups/Bows.ts

```diff
@@ -8,20 +8,20 @@ export const Bows: DisassemblySourceGroup = {
 	items: [
 		{ item: i('Shortbow'), lvl: 1 },
 		{ item: i('Longbow'), lvl: 5 },
-		{ item: i('Oak shortbow'), lvl: 10 },
-		{ item: i('Oak longbow'), lvl: 15 },
-		{ item: i('Willow shortbow'), lvl: 20 },
-		{ item: i('Willow longbow'), lvl: 25 },
-		{ item: i('Willow comp bow'), lvl: 25 },
-		{ item: i('Comp ogre bow'), lvl: 30 },
-		{ item: i('Maple shortbow'), lvl: 40 },
-		{ item: i('Maple longbow'), lvl: 45 },
-		{ item: i('Yew shortbow'), lvl: 50 },
-		{ item: i('Yew longbow'), lvl: 50 },
-		{ item: i('Yew comp bow'), lvl: 55 },
-		{ item: i('Magic shortbow'), lvl: 60 },
-		{ item: i('Magic longbow'), lvl: 65 },
-		{ item: i('Elder bow'), lvl: 70 },
+		{ item: i('Oak shortbow'), lvl: 20 },
+		{ item: i('Oak longbow'), lvl: 25 },
+		{ item: i('Willow shortbow'), lvl: 35 },
+		{ item: i('Willow longbow'), lvl: 40 },
+		{ item: i('Willow comp bow'), lvl: 45 },
+		{ item: i('Comp ogre bow'), lvl: 50 },
+		{ item: i('Maple shortbow'), lvl: 55 },
+		{ item: i('Maple longbow'), lvl: 60 },
+		{ item: i('Yew shortbow'), lvl: 70 },
+		{ item: i('Yew longbow'), lvl: 75 },
+		{ item: i('Yew comp bow'), lvl: 75 },
+		{ item: i('Magic shortbow'), lvl: 80 },
+		{ item: i('Magic longbow'), lvl: 85 },
+		{ item: i('Elder bow'), lvl: 99 },
 
 		{ item: i('Dark bow'), lvl: 90 },
```

Comment:
dark bow should probably be 99 too, if elder bow is 99

Reply:
Done